openwillis/measures/text/util/characteristics_util.py

```diff
@@ -1055,346 +642,318 @@ def update_summ_df(
 
     ------------------------------------------------------------------------------------------------------
     """
-    if len(phrase_df) > 0:
-        speech_minutes = phrase_df[measures["phrase_minutes"]].sum()
-    else:
-        speech_minutes = (float(df_diff.iloc[-1][time_index[1]]) - float(df_diff.iloc[0][time_index[0]])) / 60
-
+    speech_minutes = (float(df_diff.iloc[-1][time_index[1]]) - float(df_diff.iloc[0][time_index[0]])) / 60
```

Comment:
Do something like:

```suggestion
    if len(turn_df) > 0:
        speech_minutes = turn_df[measures["turn_minutes"]].sum()
    else:
        speech_minutes = (float(df_diff.iloc[-1][time_index[1]]) - float(df_diff.iloc[0][time_index[0]])) / 60
  ```
Otherwise if multiple speakers counts the times when the other speaking in between turns as well

Reply:
updated